modules/auxiliary/scanner/emby_ssrf_scanner.rb

```diff
@@ -0,0 +1,105 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize
+    super(
+      'Name' => 'Emby SSRF HTTP Scanner',
+      'Description' => 'Generates a `GET` request to the provided web servers and executes an SSRF against the targeted EMBY server. Returns the server header, HTML title attribute and location header (if set). This is useful for rapidly identifying web applications on the internal network using the Emby SSRF vulnerability (CVE-2020-26948).',
+      'Author' => 'Btnz',
+      'License' => MSF_LICENSE,
+      'Disclosure Date' => 'September 1 2020',
+      'References' =>
+              [
+                ['CVE', '2020-26948'],
+                ['URL', 'https://github.com/btnz-k/emby_ssrf']
+              ]              
+    )
+
+    deregister_options('VHOST', 'RPORT', 'SNAPLEN', 'SSL')
+
+    register_options(
+      [
+        OptBool.new('STORE_NOTES', [true, 'Store the information in notes. Use "notes -t http.title" to view', true]),
```

Comment:
I think you can take out the "Use" part here, it would be better in the markdown docs